src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Appearance/WindowBackgroundManager.cs

```diff
@@ -75,21 +76,22 @@ bool forceBackground
             return;
         }
 
-        _ = WindowBackdrop.RemoveBackdrop(window);
-
-        if (applicationTheme == ApplicationTheme.HighContrast)
-        {
-            backdrop = WindowBackdropType.None;
-        }
-        else
+        if(Utility.IsOSWindows11Insider1OrNewer && window.Background == Brushes.Transparent)
```

Comment:
why is this additional check needed?

Reply:
This was to allow decoupling window's background from our operations on backdrop. However I have reverted this for now. We will have to fix this, when we allow users to select the backdrop type themselves, or else binding will remain broken for window background